react-app/src/features/faq/LegacyFaq.tsx

```diff
@@ -18,6 +19,15 @@ export const LegacyFaq: React.FC<LegacyFaqProps> = () => {
   const { id } = useParams<{ id: string }>();
 
   const [openItems, setOpenItems] = useState<string[]>([]);
+
+  const expandAll = useCallback(() => {
+    const allIds = [];
+    oneMACFAQContent.forEach(({ qanda }) => {
+      qanda.forEach(({ anchorText }) => allIds.push(anchorText));
+    });
+    setOpenItems(allIds);
+  }, [setOpenItems]);
```

Comment:
[No need to wrap it](https://kentcdodds.com/blog/usememo-and-usecallback) in a `useCallback`

Reply:
I'd say, all the `useMemo` `useCallback` hooks were unnecessary in this PR